sympy/tensor/array/tests/test_arrayop.py

```diff
@@ -43,6 +43,11 @@ def test_tensorproduct():
     assert tensorproduct(A, B, a) == Array([[a*x, 2*a*x, 3*a*x], [a*y, 2*a*y, 3*a*y]])
     assert tensorproduct(B, a, A) == Array([[a*x, a*y], [2*a*x, 2*a*y], [3*a*x, 3*a*y]])
 
+    # tests for large scale sparse array
+    a = MutableSparseNDimArray.zeros(10000, 20000)
+    assert tensorproduct(a, x) == MutableSparseNDimArray.zeros(10000, 20000)
```

Comment:
Test will not pass because equality test for large scale sparse array is not fixed yet.(Referring to #16937)